torch/csrc/dynamo/python_compiled_autograd.cpp

```diff
@@ -255,6 +255,20 @@ static struct PyModuleDef _module = {
     -1,
     _methods};
 
+static py::object wrap_required_shapes(AutogradCompilerCall& compiler_call) {
+  // see [Note: Required Shapes]
+  std::vector<std::vector<c10::SymInt>> shapes;
```

Comment:
Reserve capacity please! Especially since the growth factor of std::vector is so poorly chosen iirc :(

Reply:
Perf of this code doesn't matter, since it is not on the critical path.

Reply:
Okay but theres no downside?